src/components/UserPathLog/Resetpw.js

```diff
@@ -1,151 +1,151 @@
-import React, { useState } from 'react';
-import axios from 'axios';
-import './ResetPw.css';
-// import Login from './Login';
-import { Link } from 'react-router-dom';
-import { CirclesBackground } from '../styleElements/CirclesBackground';
-import { Logo } from '../styleElements/icons/Logo';
```

Comment:
I commented out this entire component because it was throwing some error messages and not being used (we have a different password reset component that is being used)